src/mmap.rs

```diff
@@ -470,6 +464,15 @@ impl GuestMemory for GuestMemoryMmap {
     }
 }
 
+impl GetHostAddress for GuestMemoryMmap {
```

Comment:
The naming is a bit weird I think. How about `HostAddress`?

Reply:
Renamed as HostAddress:)